src/main/java/no/nav/veilarboppfolging/client/veilarbarena/PersonRequest.java

```diff
@@ -0,0 +1,9 @@
+package no.nav.veilarboppfolging.client.veilarbarena;
+
+import lombok.Data;
+import no.nav.common.types.identer.Fnr;
+
+@Data
+public class PersonRequest {
+	 Fnr fnr;
+}
```

Comment:
Bytt gjerne til Java Record, da dette har samme semantikk som `lombok.Data` og vi da slipper å dra inn denne avhengigheten her.

```suggestion
public record PersonRequest(
  Fnr fnr;
) {}
```